python_modules/libraries/dagster-dbt/dagster_dbt/cloud_v2/resources.py

```diff
@@ -217,20 +219,25 @@ def _fetch_workspace_data(self) -> DbtCloudWorkspaceData:
 
     def fetch_workspace_data(self) -> DbtCloudWorkspaceData:
         return DbtCloudWorkspaceDefsLoader(
-            workspace=self, translator=DagsterDbtTranslator()
+            workspace=self, translator=DagsterDbtTranslator(), select="fqn:*", exclude=""
         ).get_or_fetch_state()
 
     # Cache spec retrieval for a specific translator class.
     @lru_cache(maxsize=1)
     def load_specs(
-        self, dagster_dbt_translator: Optional[DagsterDbtTranslator] = None
+        self,
+        dagster_dbt_translator: Optional[DagsterDbtTranslator] = None,
+        select: str = "fqn:*",
+        exclude: Optional[str] = None,
```

Comment:
why are these set to two different values in different places? Use of empty string vs none feels inconsistent here. We should have a "default" constant that we reference everywhere; and it should probably only be set at the top level

Reply:
Yeah I agree. I updated the code to use default constants. Since all these APIs are user-facing, I used the defaults in all of them, but not in the state back defs, where select and exclude must be passed by `load_specs`.

```
DBT_CLOUD_DEFAULT_SELECT = "fqn:*"
DBT_CLOUD_DEFAULT_EXCLUDE = ""
```